src/AbstractDatadirTestCase.php

```diff
@@ -138,16 +138,16 @@ protected function assertMatchesSpecification(
             $this->assertNotSame(0, $runProcess->getExitCode(), 'Exit code should have been non-zero');
         }
         if ($specification->getExpectedStdout() !== null) {
-            $this->assertSame(
-                $specification->getExpectedStdout(),
-                $runProcess->getOutput(),
+            $this->assertStringMatchesFormat(
```

Comment:
`stdout` a `stderr` porovnavame cez **`assertStringMatchesFormat`**.
Okrem toho je tu pridany aj **`trim`**, kedze synchronne akcie casto neobsahuju `\n` na konci vystupu a IDEcko obycajne prida `\n` do `expected-stdout` a potom sa to tazko debuguje, preco to spadlo.